package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
     "build": "run-p --print-label build:freighter-api build:docs build:extension",
-    "build:netlify": "cd docs && yarn --legacy-peer-deps && yarn build",
+    "build:netlify": "cd docs && npm pkg set 'dependencies.@stellar/freighter-api'='latest' && yarn --legacy-peer-deps && yarn build",
+    "build:netlify-preview": "yarn && yarn build:freighter-api && cd docs && yarn --legacy-peer-deps && yarn build",
```

Comment:
in preview, let's have yarn workspaces sym link the local version of freighter-api so we can test out unpublished changes